jest.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  verbose: true,
```

Comment:
Maybe set this to `false`, cause it's know to discard `console.log` in tests and code, which will be confusing for the developer.